parser/proto.y

```diff
@@ -613,16 +613,17 @@ compactOptionDecls : compactOption {
 		$$ = $1
 	}
 
-compactOption: optionName '=' optionValue {
+compactOption : optionName '=' optionValue {
 		optName := ast.NewOptionNameNode($1.refs, $1.dots)
 		$$ = ast.NewCompactOptionNode(optName, $2, $3)
-	} |
-	optionName {
+	}
+	|	optionName {
```

Comment:
for consistency with everything else, I think this wants to be a space, not a tab
```suggestion
	| optionName {
```